docs/source/install/instructions/spack.rst

```diff
@@ -24,12 +24,12 @@ First `install spack <http://spack.readthedocs.io/en/latest/getting_started.html
    # get spack
    git clone https://github.com/spack/spack.git $HOME/src/spack
 
+   # build spack's dependencies via spack :)
+   $HOME/src/spack/bin/spack bootstrap
+
    # activate the spack environment
    source $HOME/src/spack/share/spack/setup-env.sh
 
-   # build spack's dependencies via spack :)
-   spack bootstrap
-
    # install a supported compiler
    spack compiler list | grep gcc@7.3.0 | spack install gcc@7.3.0 && spack load gcc@7.3.0 && spack compiler add
```

Comment:
should be
```suggestion
   spack compiler list | grep gcc@7.3.0 1>/dev/null && spack install gcc@7.3.0 && spack load gcc@7.3.0 && spack compiler add
```

Reply:
Looks good and seems to fix the above `echo` counter-example :)